news/10721.feature.rst

```diff
@@ -0,0 +1 @@
+Disallow use of script fixture in unit tests.
```

Comment:
This should use a `trivial` fragment instead of `feature` since the change is not user-facing.